src/oscar/apps/customer/notifications/services.py

```diff
@@ -1,18 +0,0 @@
-from oscar.core.loading import get_model
-
-Notification = get_model('customer', 'Notification')
-
-
-def notify_user(user, subject, **kwargs):
```

Comment:
Can we deprecate these instead of removing, and change them to call the new dispatcher methods?

Reply:
Deprecation added for all methods in `src/oscar/apps/customer/notifications/services.py`:
* `notify_user`
* `notify_users`